src/app/commands/create.ts

```diff
@@ -2,12 +2,14 @@ import { createCommand } from "commander";
 import { LiffApiClient } from "../../api/liff.js";
 import { resolveChannel } from "../../channel/resolveChannel.js";
 
-const createAction = async (options: {
+export type CreateAppOptions = {
```

Comment:
```suggestion
type CreateAppOptions = {
```

I don’t think we need to export this type because it isn’t imported anywhere else.